docs/source/installation/methods.rst

```diff
@@ -0,0 +1,67 @@
+Installation methods
+====================
+
+The following are all of the available options for the installing Ceph through different channels.
+We support 3 main installation methods, all managed by the ``ceph_origin`` variable:
+
+- ``repository``: means that you will get ceph installed through a new repository. Later below choose between 'community', 'rhcs' or 'dev'.
```

Comment:
could you please use double back ticks for ``community``, ``rhcs`` and ``dev``. You might also want to mention that you'd later choose between these options for ``ceph_repository``

Reply:
It's described below in the section for repository but I'll add it here also.